project/settings/package.mill.scala

```diff
@@ -363,20 +354,21 @@ trait CliLaunchers extends SbtModule { self =>
       buildHelperImage()
       Some(dockerParams)
     }
-    def buildHelperImage = T {
+    def buildHelperImage: Target[Unit] = Task {
       os.proc("docker", "build", "-t", Docker.customMuslBuilderImageName, ".")
-        .call(cwd = os.pwd / "project" / "musl-image", stdout = os.Inherit)
+        .call(cwd = Task.workspace / "project" / "musl-image", stdout = os.Inherit)
       ()
     }
-    override def writeDefaultNativeImageScript(scriptDest: String) = T.command {
-      buildHelperImage()
-      super.writeDefaultNativeImageScript(scriptDest)()
-    }
+    def writeNativeImageScript(scriptDest: String, imageDest: String = ""): Command[Unit] =
```

Comment:
Why not writeDefaultNativeImageScript ? Isn't that what I changed to actually be invoked?

Reply:
should be ok now